src/ebonite/build/helpers.py

```diff
@@ -45,54 +46,58 @@ def build_model_docker(image_params: Union[str, DockerImage], model: Model, serv
     return image
 
 
-def run_docker_img(container_name: str, image_params: Union[str, DockerImage],
-                   target_uri='', ports_mapping=None, detach=True):
+def run_docker_img(container_params: Union[str, DockerContainer], image_params: Union[str, DockerImage],
+                   host_params: Union[str, DockerHost] = '', ports_mapping=None, detach=True):
     """
     Runs Docker image as container
 
-    :param container_name: expected name of container to be run
+    :param container_params: expected params (or simply name) of container to be run
     :param image_params: params (or simply name) for docker image to be run
-    :param target_uri: host URI to connect to Docker daemon on, if no given "localhost" is used
+    :param host_params: host params (or simply URI) to connect to Docker daemon on, if no given "localhost" is used
     :param ports_mapping: mapping of exposed ports in container
     :param detach: if `False` block execution until container exits
     :return: nothing
     """
     if isinstance(image_params, str):
```

Comment:
for others you have separate function

Reply:
What do you mean by "others" here?

Reply:
for host and instance params

Reply:
"Others" are used in multiple methods and therefore were extracted to helpers to avoid duplication. Why would I extract 2 lines of code to separate method if it is used just once?